spec/javascript/apiService.test.js

```diff
@@ -22,12 +22,7 @@ const mockSuppAppGetFn = jest.fn(() =>
 )
 const mockGetLeaseFn = jest.fn(() => Promise.resolve({ lease: supplementalApplication.lease }))
 const mockGetStatusHistoryFn = jest.fn(() => Promise.resolve({ data: [] }))
-const mockGetUnitsFn = jest.fn(() =>
-  Promise.resolve({
-    available_units: [{ id: 'availableUnitId' }],
-    units: [{ id: 'unitId' }]
-  })
-)
+const mockGetUnitsFn = jest.fn(() => Promise.resolve({ units: [{ id: 'unitId' }] }))
```

Comment:
nit: this should just return `Promise.resolve([{ id: 'unitId' }])` directly, right? Since you removed the wrapper object

Reply:
I think it must be hitting the first promise or something weird because this works and then gives a response of [{id: 'unitId'}] that works in the test below. I was fixing it kinda late yesterday though so I'll look into it again

Reply:
Oh I see. It's because the mockGetUnitsFn is mocking the actual http request response, which gives {units:  [units]}. Then we're testing that the apiService fn calls response.units to return just [units]

Reply:
seems like all of these functions are actually poorly named and should be mockUnitsRequest or something like that.